config/routes.rb

```diff
@@ -167,7 +167,9 @@ def not_implemented
     get '/corporate/contact-us', controller: 'static_pages', action: 'show', id: 'contact-us'
     get '/corporate/cysylltu-a-ni', controller: 'static_pages', action: 'show', id: 'cysylltu-a-ni'
     resources :corporate_categories, only: [:show]
-    resources :corporate, only: [:index, :show, :create]
+    resources :corporate, only: [:index, :show, :create] do
+      get 'export_partners', on: :collection
```

Comment:
We typically use hyphens rather than underscores in URLs, although I know this isn't public facing necessarily. Just thought I'd point it out though